scripts/metadata_comparison/test/lib/helper_functions.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+def read_resource(filename):
+    path = f'test/resources/{filename}'
+    with open(path, 'r') as file:
+        data = file.read()
+    return data
```

Comment:
\n

Reply:
Thanks... I really need to find the intellij option to do this automatically for me on my new laptop!